.github/workflows/masterbot.yml

```
@@ -43,6 +43,7 @@ jobs:
           IS_PRERELEASE: 'true'
   prerelease-cdn:
     name: Pre-release CDN in dev
+    environment: 'Release'
```

`Release` requires a manual approval/stamps currently.
To isolate/segregate the usage and the other secrets, I created a new environment:

```suggestion
    environment: 'Prerelease (CDN)'
```